server/handlers/commentHandler.ts

```diff
@@ -0,0 +1,43 @@
+import { CreateCommentRequest, CreateCommentResponse, DeleteCommentRequest, DeleteCommentResponse } from '../api';
+import { db } from '../datastore';
+import { ExpressHandler, Comment } from '../types';
+import crypto from 'crypto';
+import { getUserId } from '../localStorage';
+
+//Create Comment Handler
+export const createCommentHandler
+    :ExpressHandler<CreateCommentRequest,CreateCommentResponse> = async (req,res)=>{
+
+    if(!req.body.postId)
+        return res.status(400).send({error:"No Post Id"});
+    
+    if(!getUserId())
+        return res.status(400).send({error:"No User Id"});
+    
+    if(!req.body.comment)
+        return res.status(400).send({error:"No Comment"});
+
+    const commentForInsertion:Comment = {
+        id:crypto.randomUUID(),
+        postedAt: Date.now(),
+        postId:req.body.postId,
+        userId:getUserId(),
```

Comment:
we should get this from the request for now, my next step is to extract it from the jwt token.